src/main/kotlin/pl/edu/agh/gem/external/dto/expense/ExpenseDecisionRequest.kt

```diff
@@ -0,0 +1,27 @@
+package pl.edu.agh.gem.external.dto.expense
+
+import jakarta.validation.constraints.NotBlank
+import pl.edu.agh.gem.annotation.nullorblank.NullOrNotBlank
+import pl.edu.agh.gem.internal.model.expense.Decision
+import pl.edu.agh.gem.internal.model.expense.ExpenseDecision
+import pl.edu.agh.gem.internal.validation.ValidationMessage.EXPENSE_ID_NOT_BLANK
+import pl.edu.agh.gem.internal.validation.ValidationMessage.GROUP_ID_NOT_BLANK
+import pl.edu.agh.gem.internal.validation.ValidationMessage.MESSAGE_NOT_NULL_AND_BLANK
+
+data class ExpenseDecisionRequest(
+    @field:NotBlank(message = EXPENSE_ID_NOT_BLANK)
+    val expenseId: String,
+    @field:NotBlank(message = GROUP_ID_NOT_BLANK)
+    val groupId: String,
+    val decision: Decision,
+    @field:NullOrNotBlank(message = MESSAGE_NOT_NULL_AND_BLANK)
+    val message: String?,
```

Comment:
when we have message not null and not blank we want to have `String?` ?

Reply:
changed to MESSAGE_NULL_OR_NOT_BLANK